thirdparty/zlib/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+PROJECT(zlib)
+
+if(NOT WIN32)
+            set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-all")
+            set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-all")
+endif()
+
+if (MSVC)
+  # Build with multiple processes
+  add_definitions(/MP)
+  # MSVC warning suppressions
+  add_definitions(
+    /wd4996 # The compiler encountered a deprecated declaration.
+  )
+
+endif (MSVC)
+
+set(BUILD_SHARED_LIBS OFF)
+set(TARGET_NAME "zlib")
+include_directories("${CMAKE_CURRENT_SOURCE_DIR}/zlib")
```

Comment:
`include_directories` -> `target_include_directories`

Reply:
It was changed, thanks.